package.json

```diff
@@ -20,6 +20,7 @@
     "dev": "npm run docs:dev",
     "test": "npm run test:unit",
     "test:unit": "jest --coverage",
+    "pa11y": "concurrently -k \"npm run docs:dev\" \"wait-on http://localhost:8080 && pa11y-ci --config=.pa11yci.js\"",
```

Comment:
`concurrently` lets you run multiple commands simultaneously on any platform.

`-k` will kill other processes if one exits or dies. (stop the server when `pa11y` finishes

`npm run docs:dev` the first command to run, it will spin up the docs server on `localhost:8080`

`wait-on` is the second command. It will wait for `localhost:8080` to return 200 OK then run a command

`pa11y-ci --config=.pa11yci.js` will run the CI version of Pa11y with the supplied config file.